MergeHRISClient/__init__.py

```diff
@@ -11,7 +11,7 @@
 """
 
 
-__version__ = "1.0.0"
+__version__ = "1.0.1"
```

Comment:
Right now the version is `1.2.1` according to pip. I'm not sure why this says `1.0.0` but bumping it to `1.0.1` is probably not the right number. Does it come from the config file in the backend repo?